sqlglot/__init__.py

```diff
@@ -33,38 +35,36 @@
 schema = MappingSchema()
 
 
-def parse(sql, read=None, **opts):
+def parse(sql: str, read: t.Optional[str] = None, **opts) -> t.List[t.Optional[Expression]]:
     """
-    Parses the given SQL string into a collection of syntax trees, one per
-    parsed SQL statement.
+    Parses the given SQL string into a collection of syntax trees, one per parsed SQL statement.
 
     Args:
-        sql (str): the SQL code string to parse.
-        read (str): the SQL dialect to apply during parsing
-            (eg. "spark", "hive", "presto", "mysql").
+        sql: the SQL code string to parse.
+        read: the SQL dialect to apply during parsing (eg. "spark", "hive", "presto", "mysql").
         **opts: other options.
 
     Returns:
-        typing.List[Expression]: the list of parsed syntax trees.
+        The resulting syntax tree collection.
     """
     dialect = Dialect.get_or_raise(read)()
     return dialect.parse(sql, **opts)
 
 
-def parse_one(sql, read=None, into=None, **opts):
+def parse_one(
+    sql: str, read: t.Optional[str] = None, into: t.Optional[Expression] = None, **opts
```

Comment:
into can also be a string, so you need a union

t.Optional[Expression | str] 

and add the future annotations at the top to support this